docs/source/basic/intro.rst

```diff
@@ -0,0 +1,90 @@
+Introduction
+============
+
+The *alpaka* library defines and implements an abstract interface for the *hierarchical redundant parallelism* model.
+This model exploits task- and data-parallelism as well as memory hierarchies at all levels of current multi-core architectures.
+This allows to achieve portability of performant codes across various types of accelerators by ignoring specific unsupported levels and utilizing only the ones supported on a specific accelerator.
+All hardware types (multi- and many-core CPUs, GPUs and other accelerators) are treated and can be programmed in the same way.
+The *alpaka* library provides back-ends for *CUDA*, *OpenMP*, *Boost.Fiber* and other methods.
+The policy-based C++ template interface provided allows for straightforward user-defined extension of the library to support other accelerators.
+
+The library name *alpaka* is an acronym standing for **A**\ bstraction **L**\ ibrary for **Pa**\ rallel **K**\ ernel **A**\ cceleration.
+
+Example
+-------
+
+.. literalinclude:: ../../../example/helloWorld/src/helloWorld.cpp
```

Comment:
I think it's fine for now. However, I would prefer the workshop hello world better, as it is more simple and does not have the CI workarounds.

Reply:
I agree with you. For a first look the smaller workshop example is more suitable. But then we have the problem of keeping the example up-to-date :-(

Reply:
We have discussed in a VC that we want to reduce the amount of comments and move the removed comments directly into readthedocs. We also try to remove CI dependent code. See Issue #1101